source-jdbc/src/main/java/org/apache/kylin/source/jdbc/extensible/JdbcHiveInputBase.java

```diff
@@ -101,7 +111,9 @@ protected AbstractExecutable createSqoopToFlatHiveStep(String jobWorkingDir, Str
             }
             bquery = escapeQuotationInSql(bquery);
 
-            splitColumn = escapeQuotationInSql(dataSource.convertColumn(splitColumn, FlatTableSqlQuoteUtils.getQuote()));
+            if (!isPostgresql) {
```

Comment:
Maybe we can remove this check. Because quote character can be set by `kylin.source.quote.characte`, so user can set it to blank character or double-quote character("), is it right ?